api/src/infrastructure/sql-adapter.ts

```diff
@@ -60,8 +60,7 @@ export class SQLAdapter {
       for (const value of columnValues) {
         filterClause += `(hierarchy_level_2 = '${value}' AND (`;
 
-        const filterValues = (filter.value as string).split(',');
-        for (const filterValue of filterValues) {
+        for (const filterValue of filter.values) {
```

Comment:
We are going to deprecate the sql adapter altogether, but if it makes sense for you in the current context, seems ok to be.